plugins/custom_commands.py

```diff
@@ -155,10 +157,10 @@ class CustomCommands(BasePlugin):
         else:
             t_count = len([True for i in self.ccs[gid].values() if i["author"] == msg.author.id])
 
-            if msg.author.id not in self.config_manager.config.get("bot_maintainers", []) and\
-                    not msg.author.permissions_in(msg.channel).manage_messages and\
+            if msg.author.id not in self.config_manager.config.get("bot_maintainers", []) and \
+                    not msg.author.permissions_in(msg.channel).manage_messages and \
                     t_count >= self.plugin_config[gid]["cc_limit"]:
```

Comment:
Missed this one

Reply:
That was undone by a merge conflict. Ech.